code/modules/power/monitor.dm

```diff
@@ -50,6 +50,8 @@
 	..()
 	html_machines -= src
 
+	del(interface)	//The interface has a reference to use so we delete it.
```

Comment:
Yes this is a `del()` but there's no way to not do it snoflaky without datum qdelling.